src/sagas/fetch/index.js

```diff
@@ -18,4 +18,24 @@ export function* fetchSaga(
   return null
 }
 
+export function* fetchAnythingSaga(action: Object): Generator<any, any, any> {
```

Comment:
Let us type the `action` object.

E.g. `export function* fetchAnythingSaga(action: { name: string, query: string, args: Array<string>, callback: ?Function, ... })`